recipe/meta.yaml

```
@@ -86,13 +81,12 @@ test:
     - pytest >=7.1.2
     - pytest-timeout
     - pytest-xdist
-    - matplotlib-base >=3.5.0
+    - matplotlib-base >=3.6.1
     - scikit-image >=0.19
     - pandas >=1.4.0
```

https://github.com/scikit-learn/scikit-learn/blob/1.8.0/pyproject.toml#L81

```suggestion
    - pandas >=1.5.0
```